tests/test_item.py

```diff
@@ -0,0 +1,93 @@
+                                     'category': 'none', 'status': 'pending', 'type': 'preprint'})
+        # assert success
+        self.assertEqual(response.status_code, 201)
+
+    def test_cannot_post_item_without_being_logged_in(self):
+        # log in as owner
+        self.client.logout()
+        # try to post a collection
+        response = self.client.post(reverse('collection-item-list', args=[self.collection.id]),
+                                    {'title': 't', 'description': 'd', 'source': 'mst3k', 'tags': '',
+                                     'category': 'none', 'status': 'pending', 'type': 'preprint'})
+        # assert success
+        self.assertEqual(response.status_code, 401)
+    #
+    def test_non_collection_owner_cannot_change_item_status(self):
+        self.client.login(username="rando", password="password123")
+        response = self.client.patch(reverse('collection-item-detail', args=[self.collection.id, self.item.id]),
+                                    {'status': 'rejected'})
+        self.assertNotEqual(response.status_code, 200)
+
+    # item status can be changed by people with the "approve items" permission or owners of collections
+    def collection_owner_can_change_item_status(self):
+        self.client.login(username='owner', password='password123')
+        response = self.client.patch(reverse('collection-item-detail', args=[self.collection.id], item_id=self.item.id),
+                                    {'status': 'rejected'})
+        self.assertEqual(response.status_code, 200)
+
+    def test_approve_items_permission_holder_can_change_item_status(self):
+        permissions_holder = factories.UserFactory(username="jane")
+        permissions_holder.save()
+        assign_perm("api.approve_items", permissions_holder, self.collection)
+        self.client.login(username="jane", password="password123")
+        response = self.client.patch(reverse('collection-item-detail', args=[self.collection.id, self.item.id]),
+                                     {'status': 'approved'})
+        import pprint; pprint.pprint(response.__dict__)
+        self.assertEqual(response.status_code, 200)
+
+    # item status is automatically set to approved if the collection has the "allow_all" setting set to true
+    def test_allow_all_setting_auto_accepts_new_items(self):
+        self.client.login(username="submitter", password="password123")
+        c = factories.CollectionFactory(created_by=self.owner, settings={"allow_all": "True", "collection_type": "meeting"})
+        # make new collection
+        c.save()
+        response = self.client.post(reverse('collection-item-list', args=[c.id]),
+                                    {'title': 't', 'description': 'd', 'source': 'mst3k', 'tags': '',
+                                     'category': 'none', 'status': 'pending', 'type': 'preprint'})
+        item_id = int(response.__dict__['data']['id'])
```

Comment:
Can you check `response.__dict__['data']['status']` directly instead of looking up the item and then checking its status?